docs/_data/table/master-configuration.csv

```diff
@@ -15,6 +15,7 @@ alluxio.master.lineage.recompute.interval.ms,600000
 alluxio.master.lineage.recompute.log.path,${alluxio.logs.dir}/recompute.log
 alluxio.master.port,19998
 alluxio.master.retry,29
+alluxio.master.startup.consistencycheck.enabled,true
```

Comment:
I took the approach that we would never want to branch off there, but I can go with the majority until we decide to make it consistent across the codebase.